pages/home.py

```diff
@@ -115,12 +110,17 @@ def update_output(contents, filename):
         df = parse_data(contents, filename)
         data = df.to_dict()
 
+        # Creating the dropdown menu with full labels displayed as tooltips
+        options = [{'label': x[:20] + '...' if len(x) > 20 else x, 'value': x, 'title': x} for x in df.columns]
+
         children = html.Div([
             html.P(filename, className="sidepanel_text"),
             html.Hr(),
             html.H5("Select target colum", className="sidepanel_text"),
-            dcc.Dropdown(id='column_select', className="dropdown-class",
-                         options=[{'label': x, 'value': x} for x in df.columns]),
+            dcc.Dropdown(
+                id='column_select',
```

Comment:
The `className` attribute is missing and the column select slider is plain white instead of blue. It will be fixed in my next commit.